src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -404,13 +405,6 @@ public void EnsureSolutionIsLoaded()
                 await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
 
                 await EnsureInitializeAsync();
-                var vsSolution4 = _vsSolution as IVsSolution4;
-
-                if (vsSolution4 != null)
-                {
-                    // ignore result and continue. Since results may be incomplete if user canceled.
-                    vsSolution4.EnsureSolutionIsLoaded((uint)__VSBSLFLAGS.VSBSLFLAGS_None);
-                }
```

Comment:
I assume this code has been deleted intentionally.

Reply:
Yes, see #6 in the PR description. EnsureSolutionIsLoaded has been deprecated.